tests/httpapicompact_ut/httpapicompact_ut.c

```diff
@@ -60,6 +60,14 @@ void my_gballoc_free(void* ptr)
     free(ptr);
 }
 
+int my_mallocAndStrcpy_s(char** destination, const char* source)
```

Comment:
```suggestion
int custom_mallocAndStrcpy_s(char** destination, const char* source)
```

Reply:
This is a function that has been named like that for quite a while, and used everywhere in the track 1 SDK. Changing the name would mean a huge refactoring  of the code. I think we should postpone that change.